### PR TITLE
meshlabserver: add page

### PR DESCRIPTION
**Pull request by waldyrious**
Using info from http://www.andrewhazelden.com/blog/2012/04/automate-your-meshlab-workflow-with-mlx-filter-scripts/, other pages, and the command's help

### CODE REVIEW

File: pages/common/meshlabserver.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Convert an STL file to an OBJ file:
+
+`meshlabserver -i {{input.stl}} -o {{output.obj}}`
+
+- Convert a WRL file to a OFF file, inlcuding the vertex and face normals in the output:
+
+`meshlabserver -i {{input.wrl}} -o {{output.off}} -om vn fn`
+
+- Dump a list of all the available processing filters into a text file:
```
Comment (agnivade) [inline pages/common/meshlabserver.md]: If we are not using the `.txt` extension, maybe its just better to use "into a file" ?

File: pages/common/meshlabserver.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# meshlabserver
+
+> Command line interface for the MeshLab 3D mesh processing software.
+
+- Convert an STL file to an OBJ file:
+
+`meshlabserver -i {{input.stl}} -o {{output.obj}}`
+
+- Convert a WRL file to a OFF file, inlcuding the vertex and face normals in the output:
```
Comment (agnivade) [inline pages/common/meshlabserver.md]: Can we explain what does `-om` stand for ?

Comment (waldyrious) [inline pages/common/meshlabserver.md]: Good question. I'll see if I can figure it out.

File: pages/common/meshlabserver.md (hunk 1 above)
Comment (waldyrious) [inline pages/common/meshlabserver.md]: Sure.

Comment (waldyrious) [inline pages/common/meshlabserver.md]: Huh. It seems like this is actually -o -m, at least if I'm reading this right. I'll make some tests later and update accordingly :)

File: pages/common/meshlabserver.md (hunk 2 above)
Comment (waldyrious) [inline pages/common/meshlabserver.md]: Huh. It seems like this is actually `-o -m`, at least if I'm reading [this](https://github.com/svn2github/meshlab/blob/master/meshlab/src/meshlabserver/meshlabserver.txt#L16-L27) right. I'll make some tests later and update accordingly :)

Comment (waldyrious) [inline pages/common/meshlabserver.md]: Ok, after some investigation I figured out that the `-o -m` syntax was changed form `-om` in [r6378](https://sourceforge.net/p/meshlab/code/6378/) ([11b92d3](https://github.com/svn2github/meshlab/commit/11b92d3) in the git mirror), committed on 2014-04-18 -- only days after the latest release, [v1.3.3](https://sourceforge.net/projects/meshlab/files/meshlab/)! So for now we must stick with `-om`.

The abbreviation seems to mean "output mesh (options)". Should I add "mesh" to this command description?

**Comment by waldyrious** [comment]
@agnivade updated.

**Comment by agnivade** [comment]
:+1:

**Comment by waldyrious** [comment]
Hey @agnivade or @ostera, since lgtm.co's service is gone, I've activated the feature to request reviews before PRs can be merged. Can one of you please approve the changes and squash-merge?

**Comment by ostera** [comment]
Done! :)

**Comment by waldyrious** [comment]
Thanks!

**Pull request merged**